docs/mkdocs/docs/development/quick-start.md

```
@@ -0,0 +1,205 @@
+
+Once the Dev Container is started, open a shell session in the Terminal view of VS Code. The current working directory
+will be `/workspaces/partcad` containing the source files. To install Python packages, run the following:
+
+```bash
+$ poetry install
+```
+
+It will create virtual environment in `.venv/` directory and download about 1.5 GB dependencies. Once all dependencies
+are downloaded Poetry will also install current package in editable mode, and you will see the following:
+
+```text
```

Markdownlint (0.37.0)

Expected: indented; Actual: fenced
Code block style

(MD046, code-block-style)